accord-core/src/main/java/accord/impl/AbstractConfigurationService.java

```diff
@@ -110,6 +110,11 @@ int size()
             return epochs.size();
         }
 
+        public boolean isEmpty()
+        {
+            return lastReceived == 0;
```

Comment:
we have epochs array but until its received we can't fetch it in `epochs.topologyFor`.  Can see this clearly here

```
public synchronized Topology currentTopology()
    {
        return epochs.topologyFor(epochs.lastReceived);
    }
```